### PR TITLE
Adding brief human-readable summary of our contract

### LINKED ISSUE #104: Add brief human-readable description of IP in course contract
We should add a simplified human-readable explanation of the IP portion of our contracts to http://authoring.datacamp.com, modeled on https://creativecommons.org/licenses/by-sa/4.0/.

### PR DESCRIPTION
**Pull request by gvwilson**
This will appear as a top-level heading in authoring.datacamp.com; please add lots of comments and suggestions.

Closes #104

### CODE REVIEW

File: docs/contract.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+and a *content exhibit* with project-specific details.
+This is a shorter document,
+and you will need to sign one copy for each work you do with us
+(not least because it includes delivery dates).
+
+The contract's key features are:
+
+- You grant us a perpetual non-exclusive license to use the material that you develop for us.
+  The "non-exclusive" part means that you can use the material elsewhere,
+  e.g., in other courses or workshops.
```
Comment (richierocks) [inline docs/contract.md]: Do we still ask the instructor to notify us if they do that?

File: docs/contract.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+DataCamp's contract with authors is broken into two parts to simplify future agreements:
+a *master agreement* governing our relationship,
+which you only need to sign once,
+and a *content exhibit* with project-specific details.
+This is a shorter document,
+and you will need to sign one copy for each work you do with us
+(not least because it includes delivery dates).
+
+The contract's key features are:
```
Comment (richierocks) [inline docs/contract.md]: We probably ought to mention how they get paid.

File: docs/contract.md (hunk 1 above)
Comment (gvwilson) [reply docs/contract.md]: Yes, but I thought that adding this would be getting into the weeds - happy to do so.

File: docs/contract.md (hunk 2 above)
Comment (gvwilson) [reply docs/contract.md]: Er. Yes. How *do* they get paid?

Comment (ismayc) [reply docs/contract.md]: Once a quarter is as much as I know?

Comment (gvwilson) [reply docs/contract.md]: added

File: docs/contract.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+We revised our contracts early in 2018 to include more specific language about what's expected from both sides,
+such as clearer milestones and how quickly we should provide feedback on updates to your content.
+This page is a simplified human-readable summary of its terms,
+and is not a substitute for the full agreement.
+
+DataCamp's contract with authors is broken into two parts to simplify future agreements:
+a *master agreement* governing our relationship,
+which you only need to sign once,
+and a *content exhibit* with project-specific details.
+This is a shorter document,
```
Comment (ncarchedi) [inline docs/contract.md]: "This is" --> "The latter is"?

File: docs/contract.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+- It defines a timeline for delivering materials
+  (typically six months for a full course, and less for projects and other kinds of content).
+  We are happy to negotiate extensions if real life gets in the way,
+  but if it appears that your other commitments aren't going to let you complete work in a timely fashion,
+  or if the quality of what you deliver doesn't meet our standards
+  even after we have given you feedback and a chance to make corrections,
+  we may give notice that we are terminating the contract.
+
+- If updates to the material are needed,
```
Comment (ncarchedi) [inline docs/contract.md]: "are needed" --> "are needed after launch"

File: docs/contract.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  the contract requires us to contact you and give you first option to make the required changes.
+  If you choose not to do so,
+  we may bring in another instructor to do so.
```
Comment (ncarchedi) [inline docs/contract.md]: "we may bring in another instructor and allocate a reasonable portion of future royalties to them"

File: docs/contract.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+- Liability is limited in both directions.
+
+- You are paid quarterly.
```
Comment (ncarchedi) [inline docs/contract.md]: I'd put this second after the license explanation. Something to the effect of:

In exchange for this license, you'll earn royalties based on how many learners complete your content. We generally prepay a portion of these royalties upon launch, then on a quarterly basis once you "earn out" of the prepayment.

File: docs/contract.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+such as clearer milestones and how quickly we should provide feedback on updates to your content.
+This page is a simplified human-readable summary of its terms,
+and is not a substitute for the full agreement.
+
+DataCamp's contract with authors is broken into two parts to simplify future agreements:
+a *master agreement* governing our relationship,
+which you only need to sign once,
+and a *content exhibit* with project-specific details.
+This is a shorter document,
+and you will need to sign one copy for each work you do with us
```
Comment (mnazary) [inline docs/contract.md]: for each work --> should this be clarified? e.g. each content project?

**Comment by richierocks** [comment]
@gvwilson Looks like this is almost read to go. IF you make the final edits, I'll get it merged.

**Comment by ncarchedi** [comment]
@richierocks @gvwilson please run this past Jelle before we set it live, just to make sure we're not saying anything we shouldn't be. cc @mnazary

**Pull request merged**